### PR TITLE
import link, fix side bar collapse issue

### LINKED ISSUE #1467: Mobile view throws "Link is not imported" and SideBar doesn't collapse
`Link` component is undefined inside `GlobalHeader` component and since it appears only in mobile view it is difficult to trace. 
Also, the `Sidebar` doesn't collapse on click.

### PR DESCRIPTION
**Pull request by aquibbaig**
Fixes #1467

### CODE REVIEW

File: web-server/v0.4/src/components/SiderMenu/index.js [hunk 1]
```diff
@@ -9,9 +9,9 @@ const SiderMenuWrapper = props => {
     <DrawerMenu
       getContainer={null}
       level={null}
-      handleChild={<i className="drawer-handle-icon" />}
+      handlechild={<i className="drawer-handle-icon" />}
       onHandleClick={() => {
-        props.onCollapse(!collapsed);
+        props.onCollapse(collapsed);
```
Comment (portante) [inline web-server/v0.4/src/components/SiderMenu/index.js]: Can you add a comment for why this is "NOT collapsed"?

File: web-server/v0.4/src/components/SiderMenu/index.js [hunk 2]
```diff
@@ -9,9 +9,9 @@ const SiderMenuWrapper = props => {
     <DrawerMenu
       getContainer={null}
       level={null}
-      handleChild={<i className="drawer-handle-icon" />}
+      handlechild={<i className="drawer-handle-icon" />}
```
Comment (portante) [inline web-server/v0.4/src/components/SiderMenu/index.js]: This seems like an interface inconsistency with the `DrawerMenu`.  Can you confirm?  The GitHub repo does not seem to have any references to `handlechild`.

Comment (aquibbaig) [reply web-server/v0.4/src/components/SiderMenu/index.js]: Yes, absolutely. There is no such method there and we do not use it in our code as well. So removing that.

File: web-server/v0.4/src/components/SiderMenu/index.js (hunk 1 above)
Comment (aquibbaig) [reply web-server/v0.4/src/components/SiderMenu/index.js]: There were certain inconsistencies in our code regarding props passed from BasicLayout on to the SiderMenu. I have made some minor refactors to it which makes it easy to predict the collapse behaviour now!

**Comment by aquibbaig** [comment]
@portante Thanks for the review. Fixed!!

**Comment by aquibbaig** [comment]
@gurbirkalsi Rebased, updated

File: web-server/v0.4/src/layouts/BasicLayout.js [hunk 3]
```diff
@@ -224,7 +224,7 @@ class BasicLayout extends React.PureComponent {
           // you will be forced to jump to the 403 interface without permission
           Authorized={Authorized}
           menuData={getMenuData()}
-          collapsed={collapsed !== true}
+          collapsed={collapsed}
```
Comment (gurbirkalsi) [inline web-server/v0.4/src/layouts/BasicLayout.js]: Can we consider reverting the default collapsed state to already collapsed? This allows for more space to view tables and data throughout the platform.

Comment (aquibbaig) [reply web-server/v0.4/src/layouts/BasicLayout.js]: @gurbirkalsi testing this out I can find that `collapsed` variable is misleading and passing `!collapsed` actually sets default state to collapsed,  unlike the other one

Comment (aquibbaig) [reply web-server/v0.4/src/layouts/BasicLayout.js]: Wait a minute, Let me test it out again though

Comment (aquibbaig) [reply web-server/v0.4/src/layouts/BasicLayout.js]: @gurbirkalsi You're right, my bad!

**Comment by gurbirkalsi** [comment]
@aquibbaig It looks like the default state for `collapsed` is still being set to false. Can you submit another PR to address this issue?

**Pull request merged**